src/components/PrimaryButton/index.js

```diff
@@ -1,5 +1,10 @@
 export default function PrimaryButton(props) {
-    return (<button className='rounded-[40px] bg-primary-light text-white font-bold hover:bg-primary-dark py-2 px-6 w-fit'>
-              <h3>{props.innerText}</h3>
-            </button>)
-  };
\ No newline at end of file
+  return (
+    <button
+      className="rounded-[40px] bg-primary-light text-white font-bold hover:bg-primary-dark py-2 px-6 w-fit"
+      style={{ maxHeight: "75px" }}
```

Comment:
I am confused as to what this actually does? I removed it and it doesn't change the button?

Reply:
I had some issues without the maxHeight styling. The button was stretched out vertically.